mlprimitives/jsons/dsp.stdDevFreqAnalysis.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "dsp.stdDevFreqAnalysis",
+    "author": "Rodrigo Diez <rjdiez@mit.edu>",
+    "documentation": "",
+    "description": "Anomalies detection in satellite telemetry data using the standard deviation of the frequency components calculated using time-dependent Fast Fourier Transform",
+    "modalities": [],
+    "primitive": "mlprimitives.dsp.freqAnalysis",
+    "fit": {
+        "method": "fitFreqStdDev",        
+        "args": [
+            {
+                "name": "X",
+                "description": "training signal",
+                "type": "ndarray"
+            }
+        ],
+        "output": [
+            {      
+                "name": "mask",
+                "descrtiption": "Mask used to detect anomalies in telemetry data",
+                "type": "ndarray"
+            }
+        ]
+    },
+
+    "produce": {
+        "method": "produceFreqCompare",
+        "args": [
+            {
+                "name": "X",
+                "description": "telemetry data vector",
+                "type": "ndarray"
+            }
+        ],
+        "output": [
+            {
+                
+                "name": "anomalies",
+                "descrtiption": "data vector consisting of anomalies detected in the input vector",
```

Comment:
1.  Description is misspelled.
2.  Please capitalize "Data vector...."  to make it more professional.  Please do this to all descriptions, I won't comment on each individual one.

Reply:
This JSON file is not used anymore, I'll remove it.